compiler/rustc_mir/src/const_eval/eval_queries.rs

```diff
@@ -225,6 +226,13 @@ fn validate_and_turn_into_const<'tcx>(
         let err = ConstEvalErr::new(&ecx, error, None);
         err.struct_error(ecx.tcx, "it is undefined behavior to use this value", |mut diag| {
             diag.note(note_on_undefined_behavior_error());
+            diag.note(
+                &display_allocation(
```

Comment:
Looking at the output, I think we can do a bit better than the plain dump. So maybe instead of `to_string`, let's do something like

```
format!("the raw bytes of the constant ({}", display_allocation(....))
```

Reply:
Of course, this was just a first pass, making sure I was on the right path.